tests/integration/test_check_failure.py

```diff
@@ -0,0 +1,51 @@
+"""The demo build should pass 'nikola check' and fail with missing files."""
```

Comment:
Where is it tested that the demo site passes `nikola check`?

Reply:
This can be found in `integration/test_demo_build.py`.

Reply:
It isn’t obvious without reading/being aware of `test_demo_build.py`.

Reply:
I get your point and added some documentation. Let me know what you think about this!

Reply:
That’s more obvious, thanks.